html/elements/input/email.json

```diff
@@ -36,7 +36,7 @@
                 "version_added": true
               },
               "safari_ios": {
-                "version_added": "3.1",
+                "version_added": "2",
```

Comment:
The Safari data here is just true. This feature is also likely OS-dependent and needs manual testing to see what the UI looks like. Is iOS 2 based on testing, or a general 3.1 &rarr; 2 replacement?

Reply:
This one is a general replacement, as I have no way of testing old iOS versions (I've only got BrowserStack right now).  After running a quick test in iOS 3.0 and 4.0, however, I do notice that the special email keyboard doesn't show up in iOS 3.0, so it probably _was_ introduced in 3.1...

Should we just say iOS 3 for this?